### PR TITLE
Use gender-neutral pronouns

### PR DESCRIPTION
**Pull request by bensalilijames**
Because it's the right thing to do ❤️ 

All instances were in comments only, so it shouldn't change the test results (despite not having explicitly run `npm run lint` on every package changed).

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by benhjames** [comment]
Build failing due to "JavaScript heap out of memory" for the `dojo` package. Don't think there's much I can do about this...?

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -4922,7 +4922,7 @@ declare namespace chrome.platformKeys {
     }
 
     /**
-     * This function filters from a list of client certificates the ones that are known to the platform, match request and for which the extension has permission to access the certificate and its private key. If interactive is true, the user is presented a dialog where he can select from matching certificates and grant the extension access to the certificate. The selected/filtered client certificates will be passed to callback.
+     * This function filters from a list of client certificates the ones that are known to the platform, match request and for which the extension has permission to access the certificate and its private key. If interactive is true, the user is presented a dialog where they can select from matching certificates and grant the extension access to the certificate. The selected/filtered client certificates will be passed to callback.
```
Comment (couven92) [inline types/chrome/index.d.ts]: The documentation for the Chrome APIs is copied verbatim from the official documentation pages written by Google. I am hesistant to changing that. If Google changes their documentation to use gender-neutral pronouns, those changes will propagate down to these type definitions.

File: types/chrome-apps/index.d.ts [hunk 2]
```diff
@@ -11632,7 +11632,7 @@ declare class HTMLWebViewElement extends HTMLElement {
      * If the storage partition ID starts with persist: (partition='persist:googlepluswidgets'),
      * the webview will use a persistent storage partition available to all guests in the app with the same storage partition ID.
      * If the ID is unset or if there is no 'persist': prefix, the webview will use an in-memory storage partition.
-     * his value can only be modified before the first navigation, since the storage partition of an active renderer process cannot change.
+     * This value can only be modified before the first navigation, since the storage partition of an active renderer process cannot change.
```
Comment (couven92) [inline types/chrome-apps/index.d.ts]: The documentation for the Chrome APIs is copied verbatim from the official documentation pages written by Google. I am hesistant to changing that. If Google changes their documentation to use gender-neutral pronouns, those changes will propagate down to these type definitions.

**Review by fredrikhr — CHANGES_REQUESTED** [review]
The documentation for the Chrome APIs is copied verbatim from the official documentation pages written by Google. I am hesistant to changing that. If Google changes their documentation to use gender-neutral pronouns, those changes will propagate down to these type definitions.

File: types/chrome-apps/index.d.ts (hunk 2 above)
Comment (benhjames) [reply types/chrome-apps/index.d.ts]: The docs seem to already have the typo fix proposed: https://developer.chrome.com/apps/tags/webview#partition

File: types/chrome/index.d.ts (hunk 1 above)
Comment (benhjames) [reply types/chrome/index.d.ts]: The docs seem to already have the gender-neutral wording proposed: https://developer.chrome.com/extensions/platformKeys

**Review by fredrikhr — APPROVED** [review]

File: types/fhir/index.d.ts [hunk 3]
```diff
@@ -1866,7 +1866,8 @@ declare module fhir {
          */
         identifier?: Identifier;
         /**
-         * AE | PAE An adverse event is an event that caused harm to a patient,  an adverse reaction is a something that is a subject-specific event that is a result of an exposure to a medication, food, device or environmental substance, a potential adverse event is something that occurred and that could have caused harm to a patient but did not
+         * AE | PAE 
+An adverse event is an event that caused harm to a patient,  an adverse reaction is a something that is a subject-specific event that is a result of an exposure to a medication, food, device or environmental substance, a potential adverse event is something that occurred and that could have caused harm to a patient but did not
```
Comment (antoinerousseau) [inline types/fhir/index.d.ts]: Might need a newline fix

File: types/hapi/v15/index.d.ts [hunk 4]
```diff
@@ -300,7 +300,7 @@ export interface IReplyMethods {
 	proxy(/**  an object including the same keys and restrictions defined by the route proxy handler options. */
 		options: IProxyHandlerConfig): void;
 	/** Redirects the client to the specified uri. Same as calling reply().redirect(uri).
-	 he response flow control rules apply. */
+	 The response flow control rules apply. */
```
Comment (antoinerousseau) [inline types/hapi/v15/index.d.ts]: Might wanna add a `*` at the start of the line while we are at it

File: types/react-infinite-scroll-component/index.d.ts [hunk 5]
```diff
@@ -30,7 +30,7 @@ declare namespace InfiniteScroll {
         hasMore: boolean;
 
         /**
-         * This message is shown to the user when he has seen all the records which means he's at the bottom and hasMore is false
+         * This message is shown to the user when they have seen all the records which means they're at the bottom and hasMore is false
```
Comment (antoinerousseau) [inline types/react-infinite-scroll-component/index.d.ts]: "they are" could look cleaner

**Review by antoinerousseau — CHANGES_REQUESTED** [review]

**Comment by antoinerousseau** [comment]
Nice job!

**Review by Haroenv — APPROVED** [review]
thanks! (for algoliasearch package)

**Review by antoinerousseau — APPROVED** [review]
LGTM!

File: types/fhir/index.d.ts [hunk 6]
```diff
@@ -17761,8 +17762,8 @@ declare module fhir {
          * This patient is known to be an animal (non-human)
          */
         animal?: PatientAnimal;
-        /**
-         * A list of Languages which may be used to communicate with the patient about his or her health
+        /**their
```
Comment (antoinerousseau) [inline types/fhir/index.d.ts]: oops, this was added by mistake i suppose ;)

**Review by antoinerousseau — CHANGES_REQUESTED** [review]
Sorry last little artefact to remove and we are good

File: types/fhir/index.d.ts (hunk 6 above)
Comment (benhjames) [reply types/fhir/index.d.ts]: Oops! Nice catch! 🙈

**Review by antoinerousseau — APPROVED** [review]
LGTM!

**Review by gustavderdrache — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
This is great! Unfortunately I think the only way it's going to get through the CI server is if it's broken up into a few smaller PRs that touch a smaller number of packages at once.

**Comment by antoinerousseau** [comment]
This is just documentation, I'm sure someone can bypass the CI to merge this?

**Comment by RyanCavanaugh** [comment]
The way that we validate that this is just documentation changes without other side effects is the CI. Breaking the CI on DT inconveniences hundreds of people and I'm not going to risk it for the sake of making a documentation-only PR more ergonomic to get merged.

**Comment by benhjames** [comment]
Makes sense! Have split them up into two, the other part is: #39107

**Comment by RyanCavanaugh** [comment]
Greatly appreciated @benhjames

**Pull request merged**